### PR TITLE
Documentation Links Fixed

### PR DESCRIPTION
**Pull request by BerkinAKKAYA**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div align="center">
+y<div align="center">
```
Comment (mihar-22) [inline README.md:1]: Oops.

**Review by mihar-22 — CHANGES_REQUESTED** [review]

**Comment by mihar-22** [comment]
Awesome @BerkinAKKAYA thank you! Only one small issue I commented on.

**Review by mihar-22 — APPROVED** [review]

**Comment by BerkinAKKAYA** [comment]
> Awesome @BerkinAKKAYA thank you! Only one small issue I commented on.

Thank *you*! I am a bit new to pull requests, i tried but i think i couldn't properly review your request. Could you solve it if i couldn't?

**Comment by mihar-22** [comment]
Ye everything was fine but I can't merge it because you closed it, I'll reopen and merge in a second.

**Comment by BerkinAKKAYA** [comment]
Oh sorry about that 😅 Have a good day!

**Pull request merged**